### ISSUE TITLE
How to reassign a new input symbol when load symbol from json file

### ISSUE THREAD

**Issue by daiab** [apache/incubator-mxnet#8630]
the net structure load from json like this:
**old_data** = mx.sym.var('**old_data'**)
fc = mx.sym.FullyConnected(data=**old_data**, num_hidden=10)
loss = mx.sym.SoftmaxOutput(data=fc)
now, how to change this structure to this?
**new_data** = mx.sym.var('**new_data**')
fc = mx.sym.FullyConnected(data=**new_data**, num_hidden=10)
loss = mx.sym.SoftmaxOutput(data=fc)

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.
